testdata/test-data.xml

```diff
@@ -6,50 +6,45 @@
        shortcode="4123"
        default-ontology="testonto">
 
-    <!-- permissions: see https://docs.knora.org/03-apis/api-v2/reading-user-permissions/ -->
+    <!-- permissions: see https://docs-api.dasch.swiss/02-knora-ontologies/knora-base/#permissions -->
```

Comment:
I think we should link to docs.dasch.swiss, not to the repo-specific documentations, right?

Reply:
You are right. I didn't even notice that there were also repo specific documentation sites...